packages/components/src/test/instance-filter-builder/MultiTagSelect.test.tsx

```diff
@@ -4,10 +4,21 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { expect } from "chai";
+import sinon from "sinon";
+import { EmptyLocalization } from "@itwin/core-common";
+import { IModelApp } from "@itwin/core-frontend";
+import { Presentation } from "@itwin/presentation-frontend";
 import { fireEvent, render } from "@testing-library/react";
 import { MultiTagSelect } from "../../presentation-components/instance-filter-builder/MultiTagSelect";
 
 describe("MultiTagSelect", () => {
+  beforeEach(async () => {
+    const localization = new EmptyLocalization();
+    sinon.stub(IModelApp, "initialized").get(() => true);
+    sinon.stub(IModelApp, "localization").get(() => localization);
+    await Presentation.initialize();
```

Comment:
Is it necessary to initialize `Presentation`? Maybe it would be enough to stub `Presentation.localization` same as `IModelApp.localization`?

Reply:
deleted this beforeEach step in the following commit.